standard/flags.lua

```diff
@@ -228,7 +228,10 @@ Flags.readKey('Czechoslovakia') -- returns nil
 ---@param flagName string
 ---@return string?
 function Flags._convertToKey(flagName)
-	flagName = flagName:gsub(' ', ''):lower()
+	-- lowercase all unicode
+	flagName = mw.ustring.lower(flagName)
+	-- removes all accents and special characters
+	flagName = string.gsub(mw.ustring.toNFKD(flagName), '[^%l]', '')
```

Comment:
do we know how efficient/inefficient this is?

Reply:
```lua
for i = 1, 10000 do
	local key1 = Flags._convertToKey('Åland')
	local key2 = Flags._convertToKey('Germany')
end
```

if you compare this in dev and non dev you should get an approx of what it costs

you might have to adjust the limit number depending how fast/slow it is